Honor user CFLAGS and LDFLAGS.
Use PREFIX and DESTDIR variables when installing.
--- makefile.orig	2007-01-15 11:37:25.000000000 -0600
+++ makefile	2021-07-27 09:39:54.000000000 -0500
@@ -1,5 +1,6 @@
 
 CC =		gcc
-CFLAGS =	-ggdb -Wall -D'VERSION="$(VERSION)"'
+CFLAGS +=	-ggdb -Wall -D'VERSION="$(VERSION)"'
+PREFIX ?=	/usr/local
 
 PACKAGE =	pop3proxy
@@ -19,10 +20,10 @@
 install:	all
 	strip $(TARGETS)
-	cp $(TARGETS) /usr/local/sbin
-	cp *.1 /usr/local/man/man1
+	cp $(TARGETS) $(DESTDIR)$(PREFIX)/sbin
+	cp *.1 $(DESTDIR)$(PREFIX)/share/man/man1
 
 
 pop3.proxy:	$(POP3PROXY)
-	$(CC) -o $@ $+
+	$(CC) $(LDFLAGS) -o $@ $+
 
 
